sourcery_analytics/cli/data.py

```diff
@@ -7,6 +8,8 @@
 
 @dataclasses.dataclass
 class ThresholdBreach:
+    """A model describing a single threshold breach within a single method."""
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `ThresholdBreach`

Reason for update: The docstring was added to the class.

Suggested new docstring:
```
"""A model describing a single threshold breach within a single method."""
```